src/bluehawk/commands/EmphasizeCommand.ts

```diff
@@ -0,0 +1,62 @@
+import { locationFromToken } from "../parser/locationFromToken";
+import {
+  makeBlockOrLineCommand,
+  NoAttributes,
+  NoAttributesSchema,
+} from "./Command";
+
+import { removeMetaRange } from "./removeMetaRange";
+
+export interface EmphasizeRange {
+  start: {
+    line: number;
+    column: number;
+  };
+  end: {
+    line: number;
+    column: number;
+  };
+}
+
+export const EmphasizeCommand = makeBlockOrLineCommand<NoAttributes>({
+  name: "emphasize",
+  description: "highlight line(s) in formatted output",
```

Comment:
Maybe we can be more precise/helpful here? (Mentioning CLI from the library should be ok since it's just an example)
```suggestion
  description: "identify line(s) to highlight (see `bluehawk snip --format` command)",
```